rllib/examples/inference_and_serving/policy_inference_after_training_with_dt.py

```diff
@@ -0,0 +1,166 @@
+"""
```

Comment:
Can you run this example as a unittest too in the build file? look at how other scripts in the examples folder is run as unittests

Reply:
done